BEIMA.Backend.Test/StorageService/MinioStorageProviderTest.cs

```diff
@@ -0,0 +1,164 @@
+﻿using NUnit.Framework;
+using System;
+using System.IO;
+using System.Text;
+using System.Threading.Tasks;
+using BEIMA.Backend.StorageService;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Http.Internal;
+
+namespace BEIMA.Backend.Test.StorageService
+{
+    [TestFixture]
+    public class MinioStorageTest : UnitTestBase
+    {
+        private readonly IStorageProvider _storage;
+
+        public MinioStorageTest()
```

Comment:
Again, I think this should be a setup or one time setup method rather than a constructor.